subprojects/gradle/runner/device-provider/src/testFixtures/kotlin/com/avito/android/runner/devices/internal/kubernetes/FakeReservationDeploymentFactory.kt

```diff
@@ -0,0 +1,16 @@
+package com.avito.android.runner.devices.internal.kubernetes
+
+import com.avito.android.runner.devices.model.ReservationData
+import com.fkorotkov.kubernetes.apps.metadata
+import io.fabric8.kubernetes.api.model.apps.Deployment
+
+internal class FakeReservationDeploymentFactory : ReservationDeploymentFactory {
```

Comment:
`StubReservationDeploymentFactory` ?

Reply:
I consider we will use it as a Fake in the future